test/molecule-role/molecule/compose/files/docker-compose.yml

```diff
@@ -117,7 +117,7 @@ services:
     volumes:
       - "/var/run/docker.sock:/var/run/docker.sock:ro"
   stackstate-agent:
-    image: docker.io/stackstate/stackstate-agent-2-test:{{agent_current_branch}}
+    image: docker.io/stackstate/{{ agent_docker_repo }}:{{agent_current_branch}}
```

Comment:
where does this come from ?

Reply:
Correct, from here:   https://github.com/StackVista/stackstate-agent/blob/upstream-updates-7-21-STAC12712/test/molecule-role/molecule/compose/group_vars/all.yml#L7